recipes/simpleitk/build.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+CORES=$(grep -c ^processor /proc/cpuinfo 2>/dev/null || sysctl -n hw.ncpu || 1)
+
+if [ `uname` == Darwin ]; then
+    SO_EXT='dylib'
+    CMAKE_ARGS="-D CMAKE_OSX_DEPLOYMENT_TARGET:STRING=${MACOSX_DEPLOYMENT_TARGET}"
+else
+    SO_EXT='so'
```

Comment:
Please use the `SHLIB_EXT` instead. See the `openjpeg` feedstock for an example.